server/routes/index.js

```diff
@@ -1,19 +1,18 @@
 var express = require('express');
 var router = express.Router();
+var MongoClient = require('mongodb').MongoClient; //connect to database (mLab)
+require('dotenv').config();
 
-/* GET home page. */
-router.get('/', function(req, res, next) {
-  let data = {
-    greeting: 'Hello, Welcome to the USA!',
-    countryInfo: {
-      countryName: 'United States of America',
-      population: '330,000',
-      numberOfStates: 50,
-      capital: 'Washington D.C.',
-      president: 'Sunny D'
-    }
-  };
-  res.json(data);
+var db
+MongoClient.connect('mongodb://' + process.env.DB_USERNAME + ':' + process.env.DB_PASSWORD + '@ds227821.mlab.com:27821/thetravelapp', { useNewUrlParser: true }, (err, database) => {
```

Comment:
Good job on breaking out the sensitive info to ENV vars.

But you should also break out the rest(host, port, and the db name).

One easy way is you break out the whole string to an ENV var. `MONGODB_URL`